Fix: Enemy animation offset by its column

maggithor97 opened the pull request.

maggithor97 committed:
Fix: Enemy animation offset by its column

nwessman reviewed (changes requested):
Before merging I think we should fix the other animation errors in this. The animation cycle should be: 4 -> 1 -> 4 -> 2
The third frame is offset for some reason, I think easiest would be to skip that one, it seems to be the same as frame 4 but in the wrong place. 

This then reflects into the code you just changed, so it shouldn't be every other (% 2 === 0) it should be something more like: 
`for(  0 -> column.number) newAliew.sprite.nextFrame()`

nwessman commented:
![image](https://user-images.githubusercontent.com/45625500/141299243-1ee1a509-e5e2-4f0d-8e6d-4827d72d9ada.png)

4 -> 1 -> 4 -> 2

![image](https://user-images.githubusercontent.com/45625500/141299648-f80da3ab-89c6-4701-908e-9f27ffd1fc1f.png)

The animation offset by column can be very clearly seen in this screenshot of the fourth most left enemies in the game. We can see here that the first enemy is at frame 4, the second at frame 1, the third at frame 4, and the fourth one at frame 4.

nwessman committed:
Fix: Removes borders from enemy idle animations

nwessman committed:
Fix: Enemy explosion border is removed

nwessman committed:
Fix: Enemy attack animation border is removed

This is not tested but is implemented to combat the errors all the other extracting code had.

nwessman committed:
Fix: Removes the border around enemy number 3

Removes the border around all 3 animations

nwessman committed:
Fix: Enemy animation is now offset by their column number

This fixes so the animations of the enemies are based on which column they are in. Which creates a wave effect where four enemies next to each other is in different animation states.

nwessman commented:
I have now fixed the errors I reported above, this also resolves #12
